app/services/order_purchaser.rb

```diff
@@ -35,6 +35,12 @@ def purchase!
 
     Notifier.delay.order_receipt(user: order.user, order: order) if send_receipt?
     Notifier.delay.order_notification(order, order_notification_recipient) if send_facility_notification?
+
+    order.order_details.each do |order_detail|
+      next unless order_detail.product.order_notification_recipient?
+      Notifier.delay.product_order_notification(order_detail, order_detail.product.order_notification_recipient)
```

Comment:
Can you check and see how this serializes the order detail into the delayed_jobs table? If it uses globalid, then it's fine, but if not, we should probably switch to using `Notifier.product_order_notification(...).deliver_later`. And maybe switch the other notifiers as well.